lib/utils/patch.js

```diff
@@ -94,9 +94,15 @@ function createNoUnusedDisableError(ruleId, severity, message, comment) {
             clone.column = loc.start.column + 1
             clone.endLine = loc.end.line
             clone.endColumn = loc.end.column + 1
+            clone.fix = function fix(fixer) {
+                return fixer.removeRange(loc)
+            }
         } else {
             clone.endLine = comment.loc.end.line
             clone.endColumn = comment.loc.end.column + 1
+            clone.fix = function fix(fixer) {
+                return fixer.remove(comment)
```

Comment:
Please keep line breaks which were in the comment. This can make invalid syntax in semicolon-less style. For example:

```js
foo /*eslint-disable
*/ bar
↓↓↓
foo  bar
```

Reply:
@ianobermiller Thank you for the update! LGTM except this matter. Would you make the replaced text `comment.value.includes("\n") ? "\n" : ""` or something like (and, add a test case)?

Reply:
Missed that one from the original review, thanks!